cibuildwheel/util.py

```diff
@@ -352,11 +352,25 @@ def print_new_wheels(msg: str, output_dir: Path) -> Iterator[None]:
     existing_contents = set(output_dir.iterdir())
     yield
     final_contents = set(output_dir.iterdir())
-    new_contents = final_contents - existing_contents
+
+    class FileReport(NamedTuple):
+        name: str
+        size: str
+
+    new_contents = [
+        FileReport(wheel.name, f"{(wheel.stat().st_size + 1023) // 1024:,d}")
+        for wheel in final_contents - existing_contents
+    ]
+    max_name_len = max(len(f.name) for f in new_contents)
+    max_size_len = max(len(f.size) for f in new_contents)
     n = len(new_contents)
     s = time.time() - start_time
     m = s / 60
-    print(msg.format(n=n, s=s, m=m), *sorted(f"  {f.name}" for f in new_contents), sep="\n")
+    print(
+        msg.format(n=n, s=s, m=m),
+        *sorted(f"  {f.name:<{max_name_len}s} {f.size:>{max_size_len}s} kB" for f in new_contents),
```

Comment:
```suggestion
        *sorted(f"  {f.name:<{max_name_len}s}   {f.size:>{max_size_len}s} kB" for f in new_contents),
```

An extra couple spaces might help group the columns better:

```
13 wheels produced in 9 minutes:
  spam-0.1.0-cp310-cp310-macosx_10_9_universal2.whl    4 kB
  spam-0.1.0-cp310-cp310-macosx_10_9_x86_64.whl        3 kB
  spam-0.1.0-cp310-cp310-macosx_11_0_arm64.whl         3 kB
  spam-0.1.0-cp36-cp36m-macosx_10_9_x86_64.whl         3 kB
  spam-0.1.0-cp37-cp37m-macosx_10_9_x86_64.whl         3 kB
  spam-0.1.0-cp38-cp38-macosx_10_9_universal2.whl      4 kB
  spam-0.1.0-cp38-cp38-macosx_10_9_x86_64.whl          3 kB
  spam-0.1.0-cp38-cp38-macosx_11_0_arm64.whl           3 kB
  spam-0.1.0-cp39-cp39-macosx_10_9_universal2.whl      4 kB
  spam-0.1.0-cp39-cp39-macosx_10_9_x86_64.whl          3 kB
  spam-0.1.0-cp39-cp39-macosx_11_0_arm64.whl           3 kB
  spam-0.1.0-pp37-pypy37_pp73-macosx_10_9_x86_64.whl   3 kB
  spam-0.1.0-pp38-pypy38_pp73-macosx_10_9_x86_64.whl   4 kB
```